releasenotes/notes/fix-algoliasearch-tag-non-text-6a2d90b486ca2553.yaml

```diff
@@ -0,0 +1,5 @@
+---
+fixes:
+  - |
+    algoliasearch: This fix resolves an issue where non-text search query arguments caused Type Errors when being
```

Comment:
rather than `non-text`, do we just mean, "types other than string?" Or something else? Is "text" a defined term people know? If so, then this is fine.

Reply:
Yeah, this terminology is more or less synonymous with `string` for the most part, but just keeping consistent with our previous release notes when we mentioned `set_tag_str()` vs `set_tag()`, since `set_tag_str()` only accepts text (string, bytes) arguments.